### PR TITLE
Update copy to reflect current features

### PR DESCRIPTION
**Pull request by ferndot**
- Updates the copy to remove references to "checkups" and "assessments" and to follow the requirements of the PRD

### CODE REVIEW

File: lib/src/l10n/app_en.arb [hunk 1]
```diff
@@ -86,44 +106,49 @@
...
-    "description": "Explain what the app will tell the patient once they have answered the questions"
+  "symptomReportIntroStepDataContribution": "Your answers will be anonymously submitted to a secure server and used to inform COVID-19 response efforts by providing medical experts and policy makers with information and predictive models.",
+  "@symptomReportIntroStepDataContribution": {
+    "description": "Explain that the data will be submitted anonymously and used for COVID-19 response efforts"
   },
 
-  "introStepButtonStartLabel": "START CHECKUP",
-  "@introStepButtonStartLabel": {
-    "description": "Starts the checkup by showing the first checkup screen"
+  "symptomReportIntroStepStartButton": "TAP TO START",
```
Comment (gspencergoog) [inline lib/src/l10n/app_en.arb]: Maybe just "START" or "START REPORT"?

File: lib/src/l10n/app_en.arb [hunk 2]
```diff
@@ -291,36 +311,26 @@
...
+    "description": "Describes an action you can take using the app"
   },
 
   "tutorialIntroStepAidEffort": "Aid COVID-19 response efforts.",
   "@tutorialIntroStepAidEffort": {
     "description": "Describes the benefit of the app to the community."
   },
 
-  "tutorialIntroStepGetStarted": "GET STARTED",
+  "tutorialIntroStepGetStarted": "TAP HERE TO GET STARTED",
```
Comment (gspencergoog) [inline lib/src/l10n/app_en.arb]: I think "GET STARTED" is more consistent with other apps, and is consistent with the Material Design guide, which says that button labels "describe the action that will occur if a user taps a button". Presumably everyone using a smartphone knows to tap to activate a button.

File: lib/src/l10n/app_localizations_en.dart [hunk 3]
```diff
@@ -86,31 +100,35 @@ If this is all OK and you are 18 years old or older, please click \"I agree\". O
...
 
   @override
-  String get introStepAtTheEnd =>
-      'At the end, you will receive a personalized COVID-19 risk assessment and recommendations for staying healthy.';
+  String get symptomReportIntroStepDataContribution =>
+      'Your answers will be anonymously submitted to a secure server and used to inform COVID-19 response efforts by providing medical experts and policy makers with information and predictive models.';
 
   @override
-  String get introStepButtonStartLabel => 'START CHECKUP';
+  String get symptomReportIntroStepStartButton => 'TAP TO START';
```
Comment (gspencergoog) [inline lib/src/l10n/app_localizations_en.dart]: Maybe just "START"?

Comment (ferndot) [reply lib/src/l10n/app_localizations_en.dart]: Fixed!

File: lib/src/l10n/app_en.arb (hunk 2 above)
Comment (ferndot) [reply lib/src/l10n/app_en.arb]: Makes sense. Fixed!

File: lib/src/l10n/app_localizations_en.dart [hunk 4]
```diff
@@ -52,25 +52,39 @@ If this is all OK and you are 18 years old or older, please click \"I agree\". O
   String get thankYouScreenTitle => 'Thank you';
 
   @override
-  String get thankYouScreenReturnToHome => 'RETURN TO HOME';
+  String get thankYouScreenFinishButton => 'Finish';
```
Comment (gspencergoog) [inline lib/src/l10n/app_localizations_en.dart]: This should be "FINISH" to be consistent.

**Review by gspencergoog — DISMISSED** [review]

File: lib/src/l10n/app_localizations_en.dart (hunk 4 above)
Comment (ferndot) [reply lib/src/l10n/app_localizations_en.dart]: Fixed!

**Review by gspencergoog — APPROVED** [review]
Seems like a test is still failing, though.

**Comment by ferndot** [comment]
@gspencergoog that's the iOS build, which has been failing for a while.

**Pull request merged**